azure-pipelines.yml

```diff
@@ -32,27 +32,18 @@ resources:
 stages:
 - stage: Build
   pool: sonicbld
-
+  variables:
+    CACHE_MODE: rcache
+    ${{ if eq(variables['Build.SourceBranchName'], '202012') }}:
+      VERSION_CONTROL_OPTIONS: 'SONIC_VERSION_CONTROL_COMPONENTS=deb,py2,py3,web'
   jobs:
-  - template: .azure-pipelines/build-template.yml
-    parameters:
-      platform: broadcom
-      platform_short: brcm
-      cache_mode: rcache
-      sync_rpc_image: true
-
-  - template: .azure-pipelines/build-template.yml
-    parameters:
-      platform: mellanox
-      platform_short: mlnx
-      cache_mode: rcache
-      sync_rpc_image: true
-
-  - template: .azure-pipelines/build-template.yml
+  - template: .azure-pipelines/azure-pipelines-build.yml
```

Comment:
fix this first. https://github.com/Azure/sonic-buildimage/pull/7319/files#r613663606

Reply:
@liushilongbuaa @lguohan , the change looks good for both PR build and official build.

Reply:
we have checked this behavior, there is no impact on PR check.